lisa/sut_orchestrator/azure/common.py

```diff
@@ -90,6 +90,21 @@ class AzureNodeSchema:
     )
     vhd: str = ""
     nic_count: int = 1
+    data_disk_count: int = 0
+    data_disk_caching_type: str = field(
+        default=constants.DATADISK_CACHING_TYPE_NONE,
+        metadata=schema.metadata(
+            validate=validate.OneOf(
+                [
+                    constants.DATADISK_CACHING_TYPE_NONE,
+                    constants.DATADISK_CACHING_TYPE_READONLY,
+                    constants.DATADISK_CACHING_TYPE_READYWRITE,
+                ]
+            )
+        ),
+    )
+    data_disk_iops: int = 1
```

Comment:
In Azure, it can be hard code to low end one, which is good to run test.

Reply:
Sorry, I don't understand.

Reply:
set to a default one, which is testable. For example, 4G disk size with 120 iops. It's the default value.